client/pyroclient/client.py

```diff
@@ -27,6 +27,7 @@
     "get-alerts": "/alerts",
     "get-ongoing-alerts": "/alerts/ongoing",
     "get-unacknowledged-alerts": "/alerts/unacknowledged",
+    "acknowledge-alert": "/{alert_id}/acknowledge",
```

Comment:
you need to put "alerts/{alert_id}/acknowledge"

Reply:
fixed!